### PR TITLE
@comet/create-app: Add `remove-showcase` command

### PR DESCRIPTION
**Pull request by max-debug022**
Add `remove-showcase` command to remove the showcase from a Comet project.

Example usage:

```
npx @comet/create-app remove-showcase
```

### CODE REVIEW

File: scripts/remove-showcase/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    "devDependencies": {
+        "@comet/eslint-config": "^4.3.0",
+        "@typescript-eslint/eslint-plugin": "^6.4.1",
+        "@typescript-eslint/parser": "^6.0.0",
+        "eslint-config-prettier": "^6.12.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-prettier": "^3.1.4",
+        "eslint-plugin-simple-import-sort": "^5.0.3",
+        "eslint-plugin-unused-imports": "^0.1.3",
```
Comment (nsams) [inline scripts/remove-showcase/package.json:36]: please cleanup dependencies and remove those you don't directly use

**Review by nsams — APPROVED** [review]

**Review by dkarnutsch — APPROVED** [review]

**Review by nsams — DISMISSED** [review]

**Review by nsams — DISMISSED** [review]

**Review by nsams — DISMISSED** [review]

File: create-app/src/scripts/remove-showcase/src/removeShowcase.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { ESLint } from "eslint";
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import kleur from "kleur";
+import * as deleteFileOrFolder from "rimraf";
+
+function deleteFilesAndFolders(files: string[]) {
```
Comment (johnnyomair) [inline create-app/src/scripts/remove-showcase/src/removeShowcase.ts:6]: Why not use util/deleteFilesAndFolders?

**Review by johnnyomair — CHANGES_REQUESTED** [review]

**Comment by johnnyomair** [comment]
@max-debug022 please rebase

**Review by dkarnutsch — DISMISSED** [review]

**Comment by nsams** [comment]
you should also check for comet project in cwd like in removeSite

**Comment by max-debug022** [comment]
> you should also check for comet project in cwd like in removeSite

I've moved the cwdIsCometProject function inside the utils folder so it can be used for both scripts

**Review by nsams — DISMISSED** [review]

**Comment by max-debug022** [comment]
I am sorry - I ran into trouble resolving the conflicts due to some strange behaviors. By mistake, I squashed all commits. I hope this does not cause confusion. Since this is a small PR I hope you don't mind.

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]
Could you please remove the additional `src/` folder in `src/scripts/` in a follow-up PR? I believe this is a leftover from when those scripts were separate packages...

IMO `src/scripts/removeShowcase.ts` is enough (also for `removeSite.ts`)

**Pull request merged**